ngraph/frontend/frontend_manager/src/frontend_manager.cpp

```diff
@@ -32,7 +25,8 @@ class FrontEndManager::Impl
 
     FrontEnd::Ptr loadByFramework(const std::string& framework, FrontEndCapFlags fec)
     {
-        FRONT_END_ASSERT(m_factories.count(framework))
```

Comment:
Can we add new tests for all new macros?

BTW, why cannot we use nGraph macros? If I remember right, ONNX importer uses nGraph macros.

Reply:
@itikhono Can you please handle this?

Reply:
We have created a new execution hierarchy for frontends based on the ngraph classes to exactly detect the types of problems that arise and where (frontend/ngraph), the macros are also based on ngraph helpers.
I have added unit tests for part of macros, @nosovmik can we add this file to PR ngraph/test/frontend/paddlepaddle/exceptions.cpp?

Reply:
done